UtilityCSS/utility.css

```diff
@@ -0,0 +1,89 @@
+/* reset CSS */
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-size: 62.5%;
+}
+/* reset CSS End */
+
+/* flex CSS */
+
+.container-flex {
+  display: flex;
+}
+.flex-center {
+  justify-content: center;
+  align-items: center;
+}
+
+.flex-sb {
+  justify-content: space-between;
+}
+
+.flex-sa {
+  justify-content: space-around;
+}
+
+.flex-sa {
+  justify-content: space-evenly;
+}
+
+.flex-wrap {
+  flex-wrap: wrap;
+}
+
+.flex-column {
+  flex-direction: column;
+}
+
+/* flex CSS End*/
+
+/* Border and Border Radius  CSS */
+
+.border-none {
+  border: none;
+}
+
+.border-radius-50 {
+  border-radius: 50%;
+}
+
+.border-solid {
+  border: 2px solid #000000;
+}

```

Comment:
you have alerady declared this class previously in `rules.css` . If it is same you can keep it in **common css file**